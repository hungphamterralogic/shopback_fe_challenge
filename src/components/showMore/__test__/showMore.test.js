import React from 'react';
import renderer from 'react-test-renderer';
import ShowMore from '../index';

it('renders correctly', () => {
  const tree = renderer.create(<ShowMore />).toJSON();
  expect(tree).toMatchSnapshot();
});
