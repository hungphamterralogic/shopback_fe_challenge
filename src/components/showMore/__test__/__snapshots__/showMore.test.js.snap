// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ShowMore-showMoreDesktop-0-1-1"
>
  <div
    className="ShowMore-textBoxDesktop-0-1-3"
  >
    view all popular stores
  </div>
</div>
`;
